.gitignore

```diff
@@ -1 +1,14 @@
 build/
+.vscode/c_cpp_properties.json
+CMakeCache.txt
+cmake_install.cmake
+*.cmake
+CMakeFiles/
+ext/
+ext/wiiurpxlib
+ext/libiosuhax
```

Comment:
We need these submodules, things in ext/ should not be in the gitignore
If you were having trouble with cmake, try running it from inside a folder:
```shell
mkdir build && cd build
cmake -Dwhatever_it_needs ..
```
Note the `..` for parent directory. Should keep all its stuff in a clean place where you can delete or gitignore it as needed.